packages/tag/src/react/index.tsx

```diff
@@ -56,24 +56,27 @@ const styles = {
 export interface TagStatics {
   sizes: typeof vars.sizes
 }
-
-export interface TagProps
-  extends HTMLAttributes<HTMLAnchorElement & HTMLDivElement> {
+export interface BaseProps {
```

Comment:
I'd suggest `BaseTagProps` so that it's more specific and matching most components' approach.